app/views/katello/api/v2/ansible_collections/base.json.rabl

```diff
@@ -0,0 +1,8 @@
+object @resource
+
+attributes :id
+attributes :name
+attributes :namespace
+attributes :version
+attributes :checksum
+attributes :pulp_id
```

Comment:
i know we do this for other types, but i wonder if we should stop providing a pulp_id, i know i'd like to see us do that long term, maybe we can go ahead and start with ansible.

Reply:
Removed pulp_id from rabl.